app/src/main/java/com/example/kursaa/features/episodes/presentation/EpisodeFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.kursaa.features.episodes.presentation
+
+import androidx.lifecycle.observe
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.kursaa.R
+import com.example.kursaa.core.base.BaseFragment
+import kotlinx.android.synthetic.main.fragment_episode.*
+import org.koin.androidx.scope.lifecycleScope
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class EpisodeFragment : BaseFragment<EpisodeViewModel>(R.layout.fragment_episode) {
+
+    private val linearLayoutManager: LinearLayoutManager by lifecycleScope.inject()
+    private val adapter: EpisodeRecyclerViewAdapter by lifecycleScope.inject()
```

Comment:
Czy na pewno injectowanie tego adaptera działa? Z tego co widzę nie jest zdefiniowany w żadnym z modułów.

Reply:
Przyznam szczerze, że było to zrobione z myślą o wyświetlaniu danych w RecyclerView... ale chyba zabrakło mi inspiracji. ^^"